pkg/parse/run.go

```diff
@@ -218,6 +218,13 @@ func (r *reconciler) fetch(ctx context.Context) (*SourceStatus, cmpath.Absolute,
 	if newSourceStatus.Errs == nil {
 		if err := r.syncStatusClient.SetImageToSyncAnnotation(ctx, newSourceStatus.Commit); err != nil {
 			newSourceStatus.Errs = status.Append(newSourceStatus.Errs, err)
+		} else {
+			// write the commit into the read-to-render file in hydrated root
+			if opts.RenderingEnabled {
```

Comment:
nit: this `if` block can be combined with the `else` block -> `else if opts.RenderingEnabled {`